client/components/Sponsor/fake_data.js

```diff
@@ -0,0 +1,111 @@
+export const fake_data = [
+	{
+		image:"https://source.unsplash.com/280x140/?food",
+		name:"Waba Grill",
+		category: "Asian, Bowls",
+		waiting: "25-35 mins",
+		extra: "$0 min",
+		rate: 4,
+		count: 38
+	},
+	{
+		image:"https://source.unsplash.com/280x140/?food",
+		name:"Waba Grill",
+		category: "Asian, Bowls",
+		waiting: "25-35 mins",
+		extra: "$0 min",
+		rate: 4,
+		count: 38
+	},
+	{
+		image:"https://source.unsplash.com/280x140/?food",
+		name:"Waba Grill",
+		category: "Asian, Bowls",
+		waiting: "25-35 mins",
+		extra: "$0 min",
+		rate: 4,
+		count: 38
+	},
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+	// {
+	// 	image:"https://source.unsplash.com/280x140/?food",
+	// 	name:"Waba Grill",
+	// 	category: "Asian, Bowls",
+	// 	waiting: "25-35 mins",
+	// 	extra: "$0 min",
+	// 	rate: 4,
+	// 	count: 38
+	// },
+
```

Comment:
Do you want some of my dummy data for the sponsored restaurants? I have names and images for 100 of them!

Reply:
Can I have your dummy data?? Thank you for the offer...   😃

Reply:
Here you go :) https://github.com/kayjayhogan/nav-about/blob/master/database/seed.js